lklingler-rails-tasksapp/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<p id="notice"><%= notice %></p>
+
+<h1>My Tasks</h1>
+
+<ul>
+  <% @tasks.each do |task| %>
+    <% if task.completed %>
+      <li><%= link_to "[x] #{task.title}", task %></li>
+    <% else %>
+      <li><%= link_to "[ &nbsp;] #{task.title}".html_safe, task %></li>
```

Comment:
Keep in mind the danger of using `.html_safe`

https://product.reverb.com/stay-safe-while-using-html-safe-in-rails-9e368836fac1